admin/src/test/resources/application.yml

```diff
@@ -39,9 +39,11 @@ spring:
     generate-ddl: false
   mail:
     enable: ${ENABLE_EMAIL_SERVER:false}
-    sender: ${COMPREFACE_EMAIL:example@example.com}
     host: ${EMAIL_HOST:example.com}
-    port: 25
+    username: ${EMAIL_USERNAME:example@example.com}
+    from: ${EMAIL_FROM:name <example@example.com>}
+    password: ${EMAIL_PASSWORD:example}
+    port: 587
```

Comment:
Why this variables were changed in this pull request?

Reply:
Updated. It's a tests scope. Some of this parameters are needed for running tests.